requirements.txt

```diff
@@ -3,3 +3,4 @@ requests
 tqdm
 typer>=0.4.1
 typing_extensions
+pyyaml >= 6.0.1
```

Comment:
I think this should be an optional dependency as only a small percentage of users will actually use `docker.stack.config()`. We don't want to force this dependency on everyone.

Reply:
ok, pyyaml is optional in the latest commit and could be installed as extra (pip install python_on_whales[yaml])
I also had to try importing the yaml module and handle the fact it could be missing in the config() method.

I don't know if it's the better way to do it.